bin/postserve

```diff
@@ -14,6 +14,7 @@ Options:
   --mask-layer=<layer>  If tile only has this layer covering for the whole tile (e.g. 'water'),
                         and requested zoom is more than --mask-zoom parameter, returns empty result
   --mask-zoom=<z>       If --mask-layer is set, tiles will be checked for empty after given zoom [default: 8]
+  -s --serve=<port>     Return this hostname as tileserver URL in [default: localhost]
```

Comment:
missing space on this and next line caused the `default` feature not to work. Adding it makes the code below redundant.
```suggestion
  -s --serve=<port>     Return this hostname as tileserver URL in  [default: localhost]
```